<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileAesCryptClass</class>
 <widget class="QWidget" name="FileAesCryptClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件加解密</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>加密</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>源文件:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nocrypt_file"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="nocrypt_file_se">
         <property name="text">
          <string>文件选择</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="crypt_password"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="file_encrypt">
         <property name="text">
          <string>文件加密</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>解密</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>加密文件</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="crypt_file"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="crypt_file_se">
         <property name="text">
          <string>文件选择</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>密码：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="decrypt_password"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="file_decrypt">
         <property name="text">
          <string>文件解密</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="file_aes_crypt.qrc"/>
 </resources>
 <connections/>
</ui>
